e2e-tests/playwright/e2e/verify-tls-config-health-check.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { test } from "@playwright/test";
+import { Common } from "../utils/Common";
+import { kubeCLient } from "../utils/k8sHelper";
+
+export const k8sClient = new kubeCLient();
+
+test.describe
+  .serial("Verify TLS configuration with Postgres DB health check", () => {
+  const namespace = process.env.NAME_SPACE_RDS;
+  const deploymentName = "rhdh-developer-hub";
+  const secretName = "postgres-cred";
+  const hostLatest2 = Buffer.from(process.env.RDS_2_HOST).toString("base64");
+  const hostLatest3 = Buffer.from(process.env.RDS_3_HOST).toString("base64");
+
+  test("Verify successful DB connection and successful initialization of plugins with latest-1 postgres version", async ({
+    page,
+  }) => {
+    const common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test("Change the config to use the latest-2 postgres version", async () => {
+    test.setTimeout(120000);
+    const secretData = {
+      POSTGRES_HOST: hostLatest2,
+    };
+    const patch = {
+      data: secretData,
+    };
+    await k8sClient.updateSecret(secretName, namespace, patch);
+    await k8sClient.restartDeployment(deploymentName, namespace);
+  });
+
+  test("Verify successful DB connection and successful initialization of plugins with latest-2 postgres version", async ({
+    page,
+  }) => {
+    const common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test("Change the config to use the latest-3 postgres version", async () => {
+    test.setTimeout(120000);
+    const secretData = {
+      POSTGRES_HOST: hostLatest3,
+    };
+    const patch = {
+      data: secretData,
+    };
+    await k8sClient.updateSecret(secretName, namespace, patch);
+    await k8sClient.restartDeployment(deploymentName, namespace);
```

Comment:
@josephca  This will broke the other tests which is running in parallel.
Please take a look at page 20 of this document https://docs.google.com/document/d/18zz2FAb0p0luqrIlzL3UK6A86pxojRp1X4xG-boZvyI/edit?tab=t.0

Reply:
Is this only an issue if we're planning to use the instance for other tests? This test should only run on the showcase-rds-nightly namespace which only does health checks for RDS configured RHDH instances. I can change the package.json of this to only use 1 worker, or is that not enough?

Reply:
These were the logs for the instance that passed the 'nightly' job before I refactored to get it ready for merging: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/janus-idp_backstage-showcase/1798/pull-ci-janus-idp-backstage-showcase-main-e2e-tests/1849575268008595456

Reply:
> @gustavolira Is this only an issue if we're planning to use the instance for other tests? This test should only run on the showcase-rds-nightly namespace which only does health checks for RDS configured RHDH instances. I can change the package.json of this to only use 1 worker, shouldn't that be enough to let showcase-rds-nightly mimic the safety of showcase-runtime? What do you think?

got it, so if you have only one class test it's not necessary